Supplementary Files/pentaho-EMR/install-oracle-java-8.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/sh
+/*! ******************************************************************************
+ *
+ * Pentaho Data Integration
```

Comment:
Should this be PDI? Seems like not, but not sure what it should be either.

Reply:
eGood question.  It is an example for AEL, which technically is Pentaho Data Integration.  How about "Pentaho Data Integration - Examples"?